manifests/m/Mingo/Mingo/1.8.1/Mingo.Mingo.locale.en-US.yaml

```diff
@@ -0,0 +1,26 @@
+# Created with YamlCreate.ps1 v2.0.7 using InputObject 🤖 $debug=QUSU-7-2-3
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Mingo.Mingo
+PackageVersion: 1.8.1
+PackageLocale: en-US
+Publisher: Mingo
+PublisherUrl: https://github.com/mingo-app/mingo
+PublisherSupportUrl: https://github.com/mingo-app/mingo/issues
+# PrivacyUrl: 
+Author: Mingo
+PackageName: Mingo
+PackageUrl: https://www.mingo.io
+License: Copyright © 2021 Mingo
```

Comment:
A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate